lib/MusicBrainz/Server/Controller/ISRC.pm

```diff
@@ -35,9 +35,12 @@ sub show : Chained('load') PathPart('')
     my @recordings = sort_by { $_->name } $c->model('Recording')->load(@$isrcs);
     $c->model('ArtistCredit')->load(@recordings);
     $c->stash(
-        recordings => \@recordings,
         current_view => 'Node',
         component_path => 'isrc/Index.js',
+        component_props => {
+            isrcs => $isrcs,
```

Comment:
Ok, they're already in the stash so normally I would say not to serialize them twice, but the isrcs should be a small list.

Reply:
should I remove them from the stash earlier on then?

Reply:
Ah, perhaps. I thought other actions might be using them since they are set in `_load`, but it looks like this is the only action in this controller.

Reply:
Yup, which is why I didn't touch it.